company/company.pt-BR.robomarkup

```diff
@@ -0,0 +1,91 @@
+
+Eu revi o escaneamento que você me enviou, e eu tenho que dizer que eu estou um pouco constrangida.Se eu quisesse que você encontrasse metal enferrujado e detritos inúteis você poderia simplesmente ter permanecido na terra.
+
+Icenos era um embarcação em estado da arte.Onde estão todos os componentes da nave,revestimento e computadores?Onde estão os colonos??? O clima na superfície deve ser mais severo do que é esperado para ter o dilacerado dessa forma. Então não é de se admirar que você não está coletando sobreviventes.Portanto eu concordo com sua idéia de tentar e usar esse robô que você encontrou, ao invés de você.
+
+Melhor prevenir do que remediar já que não posso perder mais NENHUM de vocês nesse trimestre. Então se conecte ao seu novo melhor amigo aí e encontre uma maneira de resgatar algo através disso. Porque enquanto nós não estivermos fazendo uma grande busca,eu estou dando uma má impressão a Nigel.E se eu estou sendo observada de uma maneira negativa,você vai acabar procurando por um novo emprego.
+
+Ciao por agora,
+Judith
+
+
+#!company id{c-underground}
+# Re:Relatório de resgate em Icenos #6
+
+Finalmente algumas notícia boa,Pelo que se vê no seu scanner é uma entrada subterrânea.Eles foram para o subsolo!Eu não creio que já ouvi falar disso antes.
+
+Isso significa que os colonos NÃO foram todos congelados até a morte/carregados pelo céu pelos ventos gelados.Metais raros,transmissores...Até as próprias pessoas poderiam estar lá embaixo.Mal posso esperar para falar a Nigel.Ooh...talvez seja algo para Iris.
+
+Ou talvez algo para Martin!Você encontra algo corpulento e eu poderia estar almoçando em Titã com Grande M. Antes do fim do ano fiscal.Em frente time!
+
+Ás vezes eu invejo você,sabia.lá fora. Milhares de milhas longe de casa sem nenhuma distração da família,amigos ou atividades de lazer.Totalmente sozinho com seus pensamentos.Muitos poderiam matar por esse tipo de paz interior.Eu imagino sua experiência.
+
+Mantenha o trabalho adequado e não tolere nenhumo destrato desse robô.
+
+Judith
+
+
+#!company id{c-lower}
+# Re: Entrada para os níveis inferiores
+
+Como o relatório informa agora ficou claro que os colonos construíram um habitat funcional lá embaixo. Eu não acredito que eu tenho que te dizer o quão significante que isso é para nós. é muito significante de fato.Isso significa que colonos estão possivelmente ainda lá embaixo.Você tem que alcança-los.Tempo é essencial!
+
+Times de especialistas em recuperação já estão a caminho para assistência. - nossos melhores homens e mulheres.Mas eles vão levar tempo até chegar à você. Use o robô para encontrar um caminho para as equipes.
+
+todo o time de executivos está firme aí contigo nisso.Martin,O Comitê DRO,Grant e Jan,Nigel,Barrington e os rapazes do grupo de direção de compras,Niam,Liam e Robin.Até Pamela.Todo mundo.E todos nós simplesmente sabemos que você tem o que é necessario para trazer esses colonos para casa.
```

Comment:
should _todo_ be _Todo_ at the beginning of the paragraph?

Reply:
Yes,please